IBPSA/Fluid/HeatPumps/ModularReversible/Data/TableData2D/EN14511/Vitocal251A08.mo

```diff
@@ -0,0 +1,52 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.EN14511;
+record Vitocal251A08 "A2W Vitocal 251 by Viessmann"
+  extends GenericAirToWater(
+    dpEva_nominal=0,
```

Comment:
Enter `dpEva_nominal` and `dpCon_nominal` from the manufacturer data sheet.
If not available, set them to zero in the example; otherwise users will simulate models with these data assuming they capture the pressure drop.

Reply:
Same as above